### PR TITLE
Minor improvements to cube generation

### PR DESCRIPTION
**Pull request by keflavich**
This just uses a slightly simpler approach for making the axes

### CODE REVIEW

File: nh3_testcubes.py [hunk 1]
```diff
@@ -21,15 +21,13 @@ def generate_cubes(nCubes=100, nBorder=1, noise_rms=0.1,
...
-    xarr11 = spaxis((np.linspace(-500, 499, 1000) * 5.72e-6
-                     + nh3con.freq_dict['oneone'] / 1e9),
-                    unit='GHz',
-                    refX=nh3con.freq_dict['oneone'] / 1e9,
-                    velocity_convention='radio', refX_unit='GHz')
-    xarr22 = spaxis((np.linspace(-500, 499, 1000) * 5.72e-6
-                     + nh3con.freq_dict['twotwo'] / 1e9), unit='GHz',
-                    refX=nh3con.freq_dict['twotwo'] / 1e9,
-                    velocity_convention='radio', refX_unit='GHz')
+    xarr11 = spaxis(np.linspace(-500, 499, 1000)*u.km/u.s,
```
Comment (low-sky) [inline nh3_testcubes.py:24]: This doesn't seem quite right for units.  How about
```
(np.linspace(-500, 499, 1000)*5.72e-6 + nh3con.freq_dict['twotwo'] / 1e9)*u.GHz
```

Comment (keflavich) [reply nh3_testcubes.py:24]: Right, you want the spacing to be exactly matched to the GBT resolution?  It's just a bit nicer/easier to read in velocity units.  I would do something like:
```
channelwidth = (5.72*u.kHz / (nh3con.freq_dict['twotwo']*u.Hz)) * constants.c
xarr11 = spaxis(np.arange(-500, 500) * channelwidth, ...)
```
or something similar.  Mixing linspace with what I assume is intended to be channelwidth isn't that clear anyway.

Comment (low-sky) [reply nh3_testcubes.py:24]: Our native data products are in frequency space (so small distortions) and a 1000 km/s velocity range would be bad.  These are just supposed to be mocks of the GBT data, which are linearly sampled in frequency.

Comment (keflavich) [reply nh3_testcubes.py:24]: Yep, that's what I figured.  There's nothing numerically wrong about the previous approach, it's just somewhat clunky in that it requires multiple uses of constants.

The approach in my most recent comment uses the GBT approximate channel width, so it should be fine.  Anyway, this isn't important.

Also, I suggest _not_ merging this PR; instead, incorporate changes into #11 if at all.  I left a comment to this effect on that PR.

Comment (mcyc) [reply nh3_testcubes.py:24]: Okay, see commit cc4023f10ba4690aaf82a64268ac2412d7bf27e8.

**Comment by mcyc** [comment]
As suggested in https://github.com/GBTAmmoniaSurvey/LineFitting/pull/14/commits/847732499354b223a67fd659e4d933088f848d3e#r265669301, this pull request will not be merged.

**Pull request closed without merging**